src/mpi/mpi_caf.c

```diff
@@ -584,8 +585,6 @@ PREFIX (register) (size_t size, caf_register_t type, caf_token_t *token,
 
   if (type != CAF_REGTYPE_COARRAY_ALLOC_REGISTER_ONLY)
     {
-//      PREFIX(sync_all) (NULL,NULL,0);
-
```

Comment:
@vehre I'm a bit confused, the contents of this commit don't appear to match the description. Perhaps you forgot to `git add` some files before committing? (Also, if you didn't already know, adding th `-v` flag to `git commit` will show the diff/patch in your editor when composing the commit message...